<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinatesDialog</class>
 <widget class="QDialog" name="CoordinatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>303</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Update Coordinates</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbTranfEnable">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Enable projection</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cbProjectionsSystems">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>158</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="title">
      <string>System A</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelCoordsA">
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Coordinates</string>
        </property>
        <property name="buddy">
         <cstring>comboCoordsA</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="comboCoordsA">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLg">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Lt</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="editLtD">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelLgD">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="editLtDm">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelLgDm">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLineEdit" name="editLtDms">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="labelLgDms">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>''</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelLgRad">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Rad</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLt">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Lg</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="editLgD">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelLtD">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="editLgDm">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="labelLtDm">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>'</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLineEdit" name="editLgDms">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QLabel" name="labelLtDms">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>''</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLabel" name="labelLtRad">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Rad</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>112</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="title">
      <string>System B</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>266</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="btCalculate">
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelXCoordsB">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>x</string>
        </property>
        <property name="buddy">
         <cstring>editXCoordB</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="editXCoordB">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelYCoordB">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>y</string>
        </property>
        <property name="buddy">
         <cstring>editYCoordB</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="editYCoordB">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLgD</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onUpdateCoordsA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLgDm</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onUpdateCoordsA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLgDms</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onUpdateCoordsA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLtD</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onUpdateCoordsA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLtDm</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onUpdateCoordsA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCalculate</sender>
   <signal>clicked()</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onButtonCalculateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTranfEnable</sender>
   <signal>clicked(bool)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onTransfEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboCoordsA</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CoordinatesDialog</receiver>
   <slot>onSystemAUnitsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSystemAClicked(QModelIndex)</slot>
  <slot>onSystemBClicked(QModelIndex)</slot>
  <slot>onSystemAUnitsChanged()</slot>
  <slot>onTextDegChanged()</slot>
  <slot>onTextMinChanged()</slot>
  <slot>onTextSecChanged()</slot>
  <slot>onTextXChanged()</slot>
  <slot>onTextYChanged()</slot>
  <slot>onUpdateCoordsA()</slot>
  <slot>onButtonCalculateClicked()</slot>
  <slot>onTransfEnabled(bool)</slot>
 </slots>
</ui>
